plotly/offline/offline.py

```diff
@@ -56,20 +58,40 @@ def init_notebook_mode():
         raise ImportError('`iplot` can only run inside an IPython Notebook.')
 
     global __PLOTLY_OFFLINE_INITIALIZED
-    # Inject plotly.js into the output cell
-    script_inject = (
-        ''
-        '<script type=\'text/javascript\'>'
-        'if(!window.Plotly){{'
-        'define(\'plotly\', function(require, exports, module) {{'
-        '{script}'
-        '}});'
-        'require([\'plotly\'], function(Plotly) {{'
-        'window.Plotly = Plotly;'
-        '}});'
-        '}}'
-        '</script>'
-        '').format(script=get_plotlyjs())
+    global __PLOTLY_USE_CDN
```

Comment:
why is this a `global` variable? it looks like you're only using it in this function context